intake/service_objects/applicant_notifications.py

```
@@ -0,0 +1,178 @@
+
+    def __init__(self, submission):
+        self.sub = submission
+        # errors is a key value store of contact methods
+        # and errors for each method
+        self.errors = {}
+        self.successes = []
+        # self.contact_methods is list of usable contact
+        # methods for this submission
+        self.contact_methods = []
+        # self.contact_info holds the contact for each method
+        self.contact_info = {}
+        self.set_contact_methods()
+        self.messages = []
+
+    def get_message_accessor(self, method):
+        return self.message_accessors[method]
+
+    def filter_contact_methods(self, contact_method_keys):
+        return contact_method_keys
+
+    def set_contact_methods(self):
+        self.contact_info = \
+            dict(ContactInfoByPreferenceField().to_representation(
+                self.sub.answers))
+        self.contact_methods = self.filter_contact_methods([
+            method
+            for method, info in self.contact_info.items()
+        ])
+        self.contact_methods.sort()
+
+    def get_context(self, contact_method=None):
+        orgs = list(self.sub.organizations.all())
+        orgs = utils.sort_orgs_in_default_order(orgs)
+        organization_names = [
+            org.name for org in orgs]
+        county_names = [
+            org.county.name for org in orgs]
+        counties_applied_to = oxford_comma(county_names)
+        if len(county_names) > 1:
+            counties_applied_to += " counties"
+        else:
+            counties_applied_to += " County"
+        return dict(
+            staff_name=utils.get_random_staff_name(),
+            name=namify(self.sub.answers['first_name']),
+            county_names=county_names,
+            counties_applied_to=counties_applied_to,
+            organizations=orgs,
+            organization_names=organization_names,
+        )
+
+    def get_notification_channel(self, method):
+        return None
```

This should raise NotImplementedError rather than None